README.md

```diff
@@ -453,6 +453,9 @@ The following resources support the Config file:
 - RDS, Neptune, and Document DB Resources
     - Resource type: `rds`
     - Config key: `DBInstances`
+- RDS DB Subnet Groups
+  - Resource type: `rds`
```

Comment:
Will there be a conflict here with the `rds` resource type already in use? We might was to follow the pattern we're using with Elasticache Subnet Groups. Maybe call this `RDSSubnetGroup` or `DBSubnetGroups`?

Reply:
Hey @arsci, I believe this is okay. For instance, we have the same resource type for `rds_cluster_types.go` file as well already.

Reply:
Would it be preferable though to have it separate? If I'm understanding how this works, you would only be able to nuke parameter groups along with RDS, Neptune, and Document DB resources. So if someone wanted to just nuke the parameter groups, they'd have to explicitly exclude every database by name in the filter in order to just nuke the parameter groups, wouldn't they? That seems very risky, and I think it would be more useful to be able to nuke them independently, like we are with the elasticache parameter groups.

Reply:
Make sense. Updated!